package-lock.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "appium-chromium-driver",
-  "version": "1.3.54",
```

Comment:
do we need to keep package.lock? we have shrinkwrap though

Reply:
Maybe? I just respected existing one. Let me do the removal after this PR